ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/SourceBuilder.java

```diff
@@ -316,229 +70,176 @@ private static PhysicalSchema getPhysicalSchema(final SourceStep<?> streamSource
       final String stateStoreName,
       final PlanInfo planInfo
   ) {
-    final boolean forceChangelog = streamSource instanceof TableSource
-        && ((TableSource) streamSource).isForceChangelog();
```

Comment:
Since we're removing this from SourceBuilder (i.e., SourceBuilderV2), let's also remove it from TableSource (i.e., TableSourceV2).